lib/data_update_scripts/20200708163323_backfill_broadcastable_type_for_broadcasts.rb

```diff
@@ -0,0 +1,9 @@
+module DataUpdateScripts
+  class BackfillBroadcastableTypeForBroadcasts
+    def run
+      Broadcast.where(broadcastable_type: nil).each do |cast|
+        cast.update(broadcastable_type: broadcast.type_of)
+      end
```

Comment:
So I ran into something interesting while QAing this locally. Check out this output:

```ruby
[2] pry(main)> Broadcast.where(broadcastable_type: nil).each do |cast|
[2] pry(main)*   cast.update(broadcastable_type: broadcast.type_of)
[2] pry(main)* end
  Broadcast Load (1.2ms)  SELECT "broadcasts".* FROM "broadcasts" WHERE "broadcasts"."broadcastable_type" IS NULL
   (0.2ms)  BEGIN
  Broadcast Exists? (1.1ms)  SELECT 1 AS one FROM "broadcasts" WHERE "broadcasts"."title" = $1 AND "broadcasts"."id" != $2 AND "broadcasts"."type_of" = $3 LIMIT $4  [["title", "Welcome Notification"], ["id", 1], ["type_of", "Onboarding"], ["LIMIT", 1]]
   (2.1ms)  ROLLBACK
```

If you read the `SELECT` statement carefully, then you'll see that I had a broadcast locally with a `type_of: "Onboarding"`, which isn't a valid `broadcastable_type`. The reason that this didn't blow up for me is that you have an `update` rather than an `update!` here, so when ActiveRecord couldn't update the broadcast, it just rolled back and moved on silently.

Although I don't think this is an issue on production (from what I can tell, there aren't any Broadcasts with a `type_of: "Onboarding"`), we probably want to decide how/if we want to handle this scenario if it happens to someone else locally.

Reply:
Huh, that's strange... 🤔 

I didn't see any Broadcasts in production with `type_of: "Onboarding"` either, which is good. I'm curious about other people's thoughts on this, though. In the meantime, I've changed `update` to `update!` in the script. 

Thanks for drawing attention to this @vaidehijoshi!